plugins/modules/ecs_ecr.py

```diff
@@ -494,6 +498,11 @@ def run(ecr, params):
                     result['policy'] = policy_text
                     raise
 
+            else:
+                original_policy = ecr.get_repository_policy(registry_id, name)
+                if original_policy:
```

Comment:
In general, we want snake_dict keys. means, the output must transform via 

```py
from ansible_collections.amazon.aws.plugins.module_utils.ec2 import camel_dict_to_snake_dict
```

but in case of an iam policy, I'm not sure about it. because in many constructs, users are loading json or fill the policy via j2.  
If you want to compare the output policy, you must transfer it back to camel case if we do that.

cc @tremble @alinabuzachis @jatorcasso

Reply:
I think the bot wants me to close this item. Let me know if you want me to do something more here.

The casing discussion makes sense. Sounds like we're comfortable with the current state of the PR? Casing will match "AWS Policy normal" and it should already be a dictionary.

Thanks!